app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+
+    masks_to_buy = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
Code Style: The error message here could be more descriptive. Consider changing it to something like 'x friends need to buy masks.'